events/SignIn/PasswordRecoveryRequested .json

```diff
@@ -9,6 +9,23 @@
     "mitreAttackTechniques": [
         "T1078 - Valid Accounts"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1531 - Account Access Removal",
+            "reason": "An attacker might use the password recovery process to lock out legitimate users by changing the password, effectively removing their access."
```

Comment:
IF the user does not change the password, the account will still have access to it. Requesting a password reset does not block the access.

Reply:
Fair point. Removed.